install/docker-compose.base.yml

```diff
@@ -24,3 +24,4 @@ services:
       - FOSSOLOGY_DB_HOST=fossology-db
       - FOSSOLOGY_DB_NAME=fossology
       - FOSSOLOGY_DB_USER=fossy
+    user: fossy
```

Comment:
not if everything except composer is successful. Then are the basic files placed and the folders created but composer does not create the classloader and all dependencies.